test/functional/Java8andUp/src/org/openj9/test/nogc/TestGCPolicyNogc.java

```diff
@@ -118,7 +125,7 @@ public void testGCPolicyNogc() {
 		while (allocator.isAlive()) {
 			try {
 				count++;
-				if (count > 5) {
+				if (count > threshold) {
```

Comment:
From test correctness point of view we can replace "5" to "200" regardless of platform. The only issue with this it might increase hang detection time from 10 to 400 seconds. @llxia is it acceptable?